support-models/src/main/scala/com/gu/support/catalog/Catalog.scala

```diff
@@ -14,12 +14,15 @@ case class Catalog(
 
 object Catalog {
   lazy val productRatePlansWithPrices: List[ProductRatePlanId] = List(
+    SupporterPlus.ratePlans(PROD),
     DigitalPack.ratePlans(PROD),
     Paper.ratePlans(PROD),
     GuardianWeekly.ratePlans(PROD),
+    SupporterPlus.ratePlans(UAT),
     DigitalPack.ratePlans(UAT),
     Paper.ratePlans(UAT),
     GuardianWeekly.ratePlans(UAT),
+    SupporterPlus.ratePlans(SANDBOX),
     DigitalPack.ratePlans(SANDBOX),
     Paper.ratePlans(SANDBOX),
     GuardianWeekly.ratePlans(SANDBOX),
```

Comment:
this can be rolled up like this
```suggestion
  lazy val productRatePlansWithPrices: List[ProductRatePlanId] = for {
    product <- List(SupporterPlus, DigitalPack, Paper, GuardianWeekly)
    env <- List(PROD, UAT, SANDBOX)
    plan <- product.ratePlans(env)
  } yield plan.id
```

Reply:
Hmm, requires quite a bit more thinking but does avoid the risk of missing one out so probably a good improvement 👍